WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameStep.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+/// Site Creation: Allows creation of the site's name.
+final class SiteNameStep: WizardStep {
+    weak var delegate: WizardDelegate?
+    private let creator: SiteCreator
+
+    private(set) lazy var content: UIViewController = {
+        return SiteNameViewController(creator: creator)
+    }()
+
+    init(creator: SiteCreator) {
+        self.creator = creator
+    }
+
+    private func didSet(name: String?) {
+        if let name = name {
+            let currentTagline = creator.information?.tagLine
```

Comment:
I doubt a current tagline will be set here, but this was just being extra defensive.